etc/picongpu/aris-grnet/gpu_picongpu.profile.example

```diff
@@ -54,9 +54,9 @@ export PIC_BACKEND="cuda:35"
 # relative to the PIConGPU source code of the tool bin/pic-create.
 export PIC_SYSTEM_TEMPLATE_PATH=${PIC_SYSTEM_TEMPLATE_PATH:-"etc/picongpu/aris-grnet"}
 
-export PATH=$PATH:$PICSRC
-export PATH=$PATH:$PICSRC/bin
-export PATH=$PATH:$PICSRC/src/tools/bin
+export PATH=$PICSRC:$PATH
```

Comment:
Could you please remove this line from all files, we do not have binaries in the main directory. I do not know why we have this in at all.